Sources/XCHammer/Generator.swift

```diff
@@ -757,7 +757,7 @@ enum Generator {
     }
 
     private static func getTulsiAspectRepo() -> String {
-        return Bundle.main.bundlePath
+        return getAssetBase()
```

Comment:
Does this work with the `SPM` build still? I haven't tested it out yet, and we should make sure it's supported before merging. I thought we needed to look in `tulsi-aspects`..

Reply:
Yep it still works, manually verified and CI has verified it as well